src/client/components/time-filter-menu/time-filter-menu.tsx

```diff
@@ -285,11 +287,11 @@ export class TimeFilterMenu extends React.Component<TimeFilterMenuProps, TimeFil
 
   render() {
     const { dimension, onClose, containerStage, openOn, inside } = this.props;
-    var { tab } = this.state;
+    const { tab } = this.state;
     if (!dimension) return null;
     const menuSize = Stage.fromSize(MENU_WIDTH, 410);
 
-    var tabs = ['relative', 'specific'].map((name) => {
+    const tabs = ['relative', 'specific'].map((name) => {
```

Comment:
please change "specific" into fixed, "fixed" is much better and consistent with "copy url - fixed time" under share menu